x-pack/plugins/observability_solution/observability_onboarding/public/application/experimental_onboarding_flow.tsx

```diff
@@ -13,32 +20,72 @@ import { Footer } from './footer/footer';
 import { OnboardingFlowForm } from './onboarding_flow_form/onboarding_flow_form';
 import { Header } from './header/header';
 
+const queryClient = new QueryClient();
+
+interface Props {
+  selectedCategory?: string;
+}
+
+function OnboardingFlowPackageList({
+  selectedCategory = 'observability',
+}: Props) {
+  const availablePackages = useAvailablePackages({
+    prereleaseIntegrationsEnabled: false,
+  });
+  return (
+    <EuiPanel>
+      <PackageListGrid
+        list={availablePackages.filteredCards.filter((card) =>
+          card.categories.includes(selectedCategory)
+        )}
+        searchTerm={''}
+        setSearchTerm={() => {}}
+        selectedCategory={selectedCategory}
+        setCategory={() => {}}
+        categories={[]}
+        setUrlandReplaceHistory={() => {}}
+        setUrlandPushHistory={() => {}}
+        showSearchTools={false}
+      />
+    </EuiPanel>
+  );
+}
+
 export function ExperimentalOnboardingFlow() {
   return (
-    <>
-      <EuiPageTemplate.Section
-        paddingSize="xl"
-        css={css`
-          & > div {
-            background-image: url(${backgroundImageUrl});
-            background-position: right center;
-            background-repeat: no-repeat;
-          }
-        `}
-        grow={false}
-        restrictWidth
-      >
-        <EuiSpacer size="xl" />
-        <Header />
-      </EuiPageTemplate.Section>
-      <EuiPageTemplate.Section paddingSize="xl" color="subdued" restrictWidth>
-        <OnboardingFlowForm />
-        <EuiSpacer size="xl" />
-      </EuiPageTemplate.Section>
-      <EuiPageTemplate.Section paddingSize="xl" grow={false} restrictWidth>
-        <Footer />
-        <EuiSpacer size="xl" />
-      </EuiPageTemplate.Section>
-    </>
+    <QueryClientProvider client={queryClient}>
+      <EuiThemeProvider darkMode={false}>
```

Comment:
The observability app is already rendering a theme provider? 

Is there a reason you are rendering another one inside the experimental onboarding flow component and hard coding the theme to light mode? This will break Kibana and user settings.

Reply:
Sorry, the code on our side hasn't been finalized/is still in draft form so it's not production-worthy yet. I will work on this a bit hopefully today or tomorrow. ++ we definitely don't want to do this, I just wrote that quickly to resolve errors when initially importing the package list.

Reply:
7e699357bafb9287bbda7c1347a893c6ed92abf9